packages/glow/src/glow.js

```diff
@@ -166,7 +166,7 @@ export function parseRow(row, lang) {
   for (const el of tags) {
     const { re, shift } = el
 
-    row.replace(re, function(match, start, n) {
+    row.replace(re, function (match, start, n) {
```

Comment:
I personally prefer not to have a whitespace here. Or is this some sort of standard?